tpm_core_node:
  ros__parameters:
    robot_type: 40
    a: [0.0,64.2,305.0,0.0,0.0,0.0,0.0]
    alpha: [180.0,90.0,0.0,90.0,-90.0,-90.0,0.0]
    d: [-169.77,0.0,0.0,-222.63,0.0,36.25,0.0]
    theta: [0.0,0.0,0.0,0.0,0.0,0.0,0.0]
    theta_shift: [0.0,90.0,0.0,0.0,0.0,0.0,0.0]
    pos_limit: [120.0,90.0,50.0,80.0,120.0,170.0,0.0]
    neg_limit: [-160.0,-33.0,-130.0,-90.0,-50.0,-170.0,0.0]
    pulse_per_unit: [100.0, 100.0, 100.0, 100.0, 100.0, 100.0]
    max_xyz_jog_speed: 40.0
    max_abc_jog_speed: 20.0
    
    # mnet config
    alm_logic: 0    # 0:low active, 1:high active
    org_logic: 1    # 0:low active, 1:high active
    feedback_src: 0 # 0:external. 1:internal
    max_axes_jog_speed: [40.0, 20.0, 20.0, 40.0, 40.0, 40.0] #unit:deg
    home_mode: 1
    home_offsets: [-171.1, -39.2, 63.3, -150.8, -94.5, 145.3] #unit:deg
    home_dir: [0, 0, 1, 0, 0, 1] 